From 157d1be023f1f994f979f4fa8395120fd3d88a6e Mon Sep 17 00:00:00 2001
From: stephen hemminger <stephen@networkplumber.org>
Date: Mon, 24 Jul 2017 10:57:27 -0700
Subject: [PATCH 41/98] netvsc: prefetch the first incoming ring element

In interrupt handler, prefetch the first incoming ring element
so that it is in cache by the time NAPI poll gets to it.

Signed-off-by: Stephen Hemminger <sthemmin@microsoft.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/hyperv/netvsc.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/hyperv/netvsc.c	2017-12-05 16:21:06.077591391 -0800
+++ b/drivers/net/hyperv/netvsc.c	2017-12-05 16:21:36.841199905 -0800
@@ -30,6 +30,7 @@
 #include <linux/if_ether.h>
 #include <linux/vmalloc.h>
 #include <linux/rtnetlink.h>
+#include <linux/prefetch.h>
 #include <asm/sync_bitops.h>
 
 #include "hyperv_net.h"
